testing/integration/upgrade_retry_download_test.go

```diff
@@ -0,0 +1 @@
+package integration
```

Comment:
:thinking: 

It looks like either you forgot to commit the implementation or committed this file by accident.

Reply:
Yes, still working on this file.  Didn't mean to commit it yet.  Perils of having to move fast (what was the other half of that saying 🤔 😉?).  Will remove it for now.

Reply:
Removed in 48d46ca6e.